### PR TITLE
add tilt development env documentation

### PR DESCRIPTION
**Pull request by saweber**
Add documentation for tilt, which is already in OpenCost and helm.

If anything is unclear or missing let me know, I would love to clarify and make this better!

### CODE REVIEW

File: docs/installation/development.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+- [Helm](https://helm.sh/)
+- [kind](https://kind.sigs.k8s.io/) or your preferred tool for a kubernetes development environment
+- [kubectl](https://kubernetes.io/docs/reference/kubectl/)
+- [Node.js](https://nodejs.org/en)
+- [Tilt](https://tilt.dev)
+
+## Run OpenCost
+1. In a terminal, run: `helm repo add prometheus-community https://prometheus-community.github.io/helm-charts`
+2. Update `tilt-values.yaml` to desired attributes - [see documentation here.](https://github.com/opencost/opencost-helm-chart/blob/main/charts/opencost/README.md)
+3. Update `Tiltfile` to allow your remote kubernetes contexts, if desired. See comment block near top of the `Tiltfile`. [See documentation here.](https://docs.tilt.dev/api#api.allow_k8s_contexts)
```
Comment (mattray) [inline docs/installation/development.md:24]: And here

File: docs/installation/development.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+4. If using `kind`, in a terminal, run `kind create cluster` to create local cluster, or set your kubernetes context to the desired cluster.
```
Comment (mattray) [inline docs/installation/development.md:25]: Kubernetes

File: docs/installation/development.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+5. In a terminal, run: `tilt up -- --arch [amd64 or arm64] --docker-repo [repo or username]` at the root of the OpenCost repository
+        `--arch amd64` will compile OpenCost for amd64 nodes, `--arch arm64` will compile for arm64 nodes
+        `--docker_repo` is only required for remote development or you want the images saved in dockerhub. The value is prepended to the image tag. [See Docker Documentation](https://docs.docker.com/engine/reference/commandline/build/#tag)
```
Comment (mattray) [inline docs/installation/development.md:28]: "Docker Hub"

File: docs/installation/development.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+2. Update `tilt-values.yaml` to desired attributes - [see documentation here.](https://github.com/opencost/opencost-helm-chart/blob/main/charts/opencost/README.md)
+3. Update `Tiltfile` to allow your remote kubernetes contexts, if desired. See comment block near top of the `Tiltfile`. [See documentation here.](https://docs.tilt.dev/api#api.allow_k8s_contexts)
+4. If using `kind`, in a terminal, run `kind create cluster` to create local cluster, or set your kubernetes context to the desired cluster.
+5. In a terminal, run: `tilt up -- --arch [amd64 or arm64] --docker-repo [repo or username]` at the root of the OpenCost repository
+        `--arch amd64` will compile OpenCost for amd64 nodes, `--arch arm64` will compile for arm64 nodes
+        `--docker_repo` is only required for remote development or you want the images saved in dockerhub. The value is prepended to the image tag. [See Docker Documentation](https://docs.docker.com/engine/reference/commandline/build/#tag) 
+6. Follow instructions to open tilt UI in browser (hit space, or navigate to http://localhost:10350)
```
Comment (mattray) [inline docs/installation/development.md:29]: "Tilt"

File: docs/installation/development.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+7. Make changes, Tilt will automatically watch source code and recompile, built, deploy, and run tests.
```
Comment (mattray) [inline docs/installation/development.md:30]: "build" not "built"

**Review by mattray — COMMENTED** [review]
Nitpicky typos, the content's great though!

**Comment by saweber** [comment]
Thank you for the extra set of eyes @mattray!

**Review by mattray — APPROVED** [review]

**Comment by mattray** [comment]
Looks good!

**Pull request merged**